packages/yoroi-extension/app/ergo-connector/components/layout/Layout.js

```diff
@@ -34,21 +35,24 @@ export default class Layout extends Component<Props> {
     const { intl } = this.context;
 
     return (
-      <div className={styles.layout}>
-        <div className={styles.header}>
-          <Link to={ROUTES.SETTINGS.GENERAL} className={styles.menuIcon}>
-            <MenuIcon />
-          </Link>
-          <div className={styles.menu}>
-            <YoroiLogo />
-            <div className={styles.logo}>
-              <h3>{intl.formatMessage(messages.yoroiDappConnector)}</h3>
-              <p className={styles.poweredBy}>{intl.formatMessage(messages.poweredBy)}</p>
+      <>
+        <TestnetWarningBanner isTestnet={false} />
```

Comment:
Is there a reason you included the banner before the layout div in both of these files? If you include it before the layout, it causes a scrollbar to appear.